data/config_options.yml

```
@@ -132,6 +146,10 @@ options:
       since: 0.1.0
       type: string
       default_value: null
+    nodejs:
+      config_key: pushApiKey
```

My bad, this is actually `apiKey` according to this: https://github.com/appsignal/appsignal-nodejs/blob/a75027dacc3ba1920e579218350f64c09a606d1b/packages/nodejs/src/config/configmap.ts#L3

```suggestion
      config_key: apiKey
```